.github/ISSUE_TEMPLATE/enhancement-proposal.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Enhancement Proposal
+about: Use this template for submitting enhancement requests
+title: "[ENHANCEMENT]"
+labels: ''
```

Comment:
do we want a label for better filtering like `kind/enhancement` or `kind/feature`?

Reply:
Sure I'll add one!